lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -1,9 +1,62 @@
 """
 Progress Tab Serializers
 """
-
+import uuid
 from rest_framework import serializers
 from lms.djangoapps.course_home_api.outline.v1.serializers import CourseBlockSerializer
+from rest_framework.reverse import reverse
+
+
+class CoursewareSummarySerializer(serializers.Serializer):
+    """
+    Serializer
+    """
+    blocks = serializers.SerializerMethodField()
+
+    def get_blocks(self, blocks):
+        return [
+            {
+                'display_name': chapter['display_name'],
+                'chapter_url': reverse(  # currently points to the LMS url not the MFE
+                    'jump_to',
+                    kwargs={'course_id': str(self.context['course_key']), 'location': str(chapter['url_name'])},
+                    request=self.context['request'],
+                ),
```

Comment:
There might be a function to use from the courseware api that can tell us whether the links should be directed. Although I'm not sure you need to tackle that as part of this. That is something we will need to fix for the dates tab as well and outline tab when we get there too.